configs/gpalloc.conf.ctmpl

```diff
@@ -1,11 +1,11 @@
 {{with $environment := env "ENVIRONMENT"}}
 {{with $image := env "IMAGE"}}
-{{with $billingPemSecrets := vault (printf "secret/dsde/firecloud/%s/common/billing-account.pem" $environment)}}
-{{with $refreshTokenServiceAccount := vault (printf "secret/dsde/firecloud/%s/common/refresh-token-service-account.json" $environment)}}
+{{with $billingServiceAccount := vault (printf "secret/dsde/firecloud/%s/common/billing-account.json" $environment)}}
+{{with $refreshTokenServiceAccount := vault (printf "secret/dsde/firecloud/%s/common/refresh-token-oauth-credential.json" $environment)}}
```

Comment:
the old SA is gone (maybe it moved?). this one is analogous from what i can tell. thurloe uses it for swagger and i'm sure other services too.